internal/jsondb/db_test.go

```diff
@@ -49,7 +49,7 @@ func TestDegenerate(t *testing.T) {
 		db := jsondb.New(dir, 0755)
 
 		// write-only file
-		err = ioutil.WriteFile(path.Join(dir, "one.json"), []byte("{"), 0644)
+		err = ioutil.WriteFile(path.Join(dir, "one.json"), []byte("{"), 600)
```

Comment:
Dropping the leading `0` changes this from octal to decimal.

Reply:
I was wondering the same. Was it intentional @ondrejbudai ?

Reply:
I think that gosec didn't like `0644` and preferred `0600`. `600` is definitely a mistake.